README.md

```diff
@@ -291,7 +291,7 @@ data: [{'token_i32': 1234, 'token_i64': 645664, 'token_str': '078-05-1111', 'tok
 Read the data using the same ZoneId. AcraServer decrypts the data and returns plaintext:
 ```bash
 docker exec -it python-mysql_python_1 \
-  python3 extended_example_with_zone.py --host=acra-server --port=9393 --print --zone_id=DDDDDDDDPfBoWiixeMTUuEOk
```

Comment:
hm, why you removed zone argument?

Reply:
Hm, typo. need to delete `zone_id` from another query

Reply:
Hm, typo, need to fix execution script